games-roguelike/hengband/hengband-1.6.2-r1.ebuild

```diff
@@ -2,7 +2,7 @@
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=5
-inherit eutils autotools games
+inherit eutils autotools
```

Comment:
Sort, EAPI 7, but you know that I hope ;)

Reply:
(Especially given you're already using BDEPEND!!)

Reply:
Definitely! I've been trying to attack the updates one at a time, with the eapi upgrade scheduled next (including adding `PATCHES` variable)